### PR TITLE
Input data timestamping

### PR DESCRIPTION
**Pull request by sergiogiro**

### CODE REVIEW

File: cli/src/plz/cli/input_data.py [hunk 1]
```diff
@@ -51,10 +52,24 @@ def publish(self) -> Optional[str]:
...
         self.tarball = None
+        self.input_id = None
+        self._timestamp_millis = None
 
     def __enter__(self):
+        # Try to avoid building the tarball. Look at maximum modification
+        # time in the input, and if we have in input for the timestamp, use
+        # that one
+        input_id = self._get_input_from_controller_or_none().get('id', None)
+        log_debug(f'Input id from the controller: {input_id}')
```
Comment (SamirTalwar) [inline cli/src/plz/cli/input_data.py]: “ID” plz.

File: test/run [hunk 2]
```diff
@@ -293,6 +294,11 @@ function redact_instance_status {
   grep -v '^Instance status: '
 }
 
+function redact_debug_messages {
+  grep -v '^:: '
+}
```
Comment (SamirTalwar) [inline test/run]: Why is this needed? Tests don’t run in debug mode, do they?

Comment (sergiogiro) [reply test/run]: For your reference, plz find below the definition of `log_debug`:
```
def log_debug(message):
    print('::', message)
```
